simulariumio/filters/translate_filter.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Dict
+import logging
+
+import numpy as np
+
+from .filter import Filter
+from ..data_objects import TrajectoryData
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class TranslateFilter(Filter):
```

Comment:
I'm feeling like you need a TransformFilter(translate:vec3=[0,0,0], rotate:vec3=[0,0,0], scale:vec3=[1,1,1]) that can do any arbitrary transform applied in `T * R * S * position` order. (scale first, then rotate, then translate)

That could take the place of a few of the specialized filters you have

Reply:
Or even alternatively accept a ready made 4x3 or 4x4 matrix

Reply:
yeah I should do that. I still need to implement arbitrary rotation, but I'll make a note to combine them when I do that